addons/website_slides/models/slide_slide.py

```diff
@@ -1042,6 +1083,72 @@ def _fetch_google_drive_metadata(self, fetch_image, return_when_not_found=False)
 
         return slide_metadata
 
+    def _fetch_vimeo_metadata(self, fetch_image=True, return_when_not_found=False):
```

Comment:
Same remarks for parameters + return type, if we update other methods, just propagate here :) .